pkgs/by-name/ob/obs-cmd/package.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "obs-cmd";
+  version = "0.17.4";
+
+  src = fetchFromGitHub {
+    owner = "grigio";
+    repo = pname;
```

The general sentiment is to avoid re-using `pname` (see #277994 for details).
```suggestion
    repo = "obs-cmd";
```